src/component/teacher/Teacher.js

```diff
@@ -0,0 +1,83 @@
+import { useState, useEffect } from "react";
+import RenderLesson from "./RenderLesson";
+import Form from "./Form";
+import AddQuestion from "./AddQuestion";
+import axios from "axios";
+
+
+function Teacher() {
+  const [visible, setVisible] = useState(false);
+  const [lessons, setLessons] = useState([]);
+ 
+
```

Comment:
Delete line 12, I only want one space between const and useEffect.

Reply:
Thank you @douglas86. Done!